### ISSUE TITLE
OCP4 Migration - Deploy SSO initial instances

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/ocp-sso#147]
**Describe the issue**
After the pipeline is migrated, we will need to deploy a initial instance of sso in each respective environment. This will require modification of the infra code for things like imagestreams and storage classes maybe. 

**Definition of done**
*SSO initial instance deployment*
- [x] sandbox instance deployed
- [x] dev instance deployed
- [x] test deployed
- [x] prod deployed
- [x] infrastructure code updated

Here are the detailed components to spin up and initialize:
*SSO components*
- [x] import redhat sso image
- [x] build bcgov patroni v10 image
- [x] migrate secrets
- [x] deploy patroni statefulset with netapp-block (this is not yet tested in ocp3)
- [x] deploy sso ap

*ops components*
- [x] bcdk pipeline
- [ ] database backup container (covered in another ticket)
- [x] audit logs backup
- [x] k6 testing

**Comment by ShellyXueHan** [comment]
### Update:
Here is to track the verification steps needed before we can kick off pipeline job:
- [x] import redhat sso image
- [x] build bcgov patroni v10 image
- [x] migrate secret templates
- [ ] deploy patroni statefulset with netapp-block (this is not yet tested in ocp3)
- [ ] deploy sso app
- [ ] create routes with cert

**Comment by ShellyXueHan** [comment]
### Update:
https://github.com/bcgov/ocp-sso/pull/148
